frontend/src/constants/pages.ts

```diff
@@ -1,7 +1,7 @@
 export const PAGES = {
   profile: 'profile',
-  coursesList: 'coursesList',
-  myCourses: 'myCourses',
+  coursesList: 'courses-list',
+  myCourses: 'my-courses',
```

Comment:
Please use common code styles, either rename all the pages to camelCase or via_underscore

Reply:
why does name of pages in url go with "-" so

Reply:
cause they are defined in PATHS, not PAGES